app/controllers/feedback_controller.rb

```diff
@@ -64,14 +59,17 @@ def after_create_url
 
   def feedback_params
     params.require(:feedback).permit(
-      :type,
-      :comment,
-      :rating,
+      :task, :rating, :comment, :other_reason, :type,
       :event,
       :outcome,
       :case_number,
       :referrer,
-      :email
+      :email,
+      reason: []
     )
   end
+
+  def setup_page
+    @feedback_form = FeedbackForm.new if type == 'feedback'
```

Comment:
The `eql?` operator is the same as `==`, but which should be used for our standards?

Reply:
I prefer the `eql?` method, but not a big deal IMO